Gems/Atom/RHI/Metal/Code/Source/RHI/Device.cpp

```diff
@@ -59,7 +59,7 @@ namespace AZ
                 releaseQueueDescriptor.m_collectLatency = m_descriptor.m_frameCountMax;
                 m_releaseQueue.Init(releaseQueueDescriptor);
             }
-             
```

Comment:
Unnecessary change?

Reply:
Ah, it is the trim trailing whitespace on save option in VSCode